projects/plugins/jetpack/modules/widget-visibility/editor/index.jsx

```diff
@@ -0,0 +1,385 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment, useEffect, useCallback, useMemo } from '@wordpress/element';
+import { BaseControl, Button, SelectControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InspectorAdvancedControls } from '@wordpress/block-editor'; // eslint-disable-line import/no-unresolved
```

Comment:
I must admit that it took me a few seconds to understand where the settings were in the UI. I expected a new panel, as I've learnt to ignore the "Advanced" panel since I've hardly ever had the use for it since I started using the block editor.

That's just my personal experience though :)

Reply:
I was going off Matias' comment in the original issue, however I'd be happy to make it a new panel if that's something that's easily done, do we have any examples of doing that elsewhere?

Reply:
We rely on `InspectorControls` for quite a few of our blocks, but I don't believe we have any example of InspectorControls added to existing core blocks at the moment.

I'm happy to defer to @mtias on this though; I'm not Gutenberg UI expert :)

Reply:
Correct, this feature must be turned on in jetpack settings:
![2021-08-30_10-36](https://user-images.githubusercontent.com/937354/131365212-900d8a03-9839-44f5-bd59-f477d6baad30.png)